src/libs/antares/study/parameters.cpp

```diff
@@ -1876,35 +1873,23 @@ void Parameters::saveToINI(IniFile& ini) const
 
     // Variable selection
     {
-        assert(!variablesPrintInfo.isEmpty());
         uint nb_tot_vars = (uint)variablesPrintInfo.size();
-        uint selected_vars = 0;
+        uint nb_selected_vars = (uint)variablesPrintInfo.namesOfEnabledVariables().size();
```

Comment:
You create a `vector` here, but use only its size. I suggest creating a dedicated method with a simple implementation (using e.g `std::count_if`) instead.